packages/vendure-plugin-invoices/src/invoice.plugin.ts

```diff
@@ -84,7 +84,7 @@ export interface InvoicePluginConfig extends InvoicePluginConfigInput {
   },
   compatibility: '>=2.2.0',
   configuration: (config: RuntimeVendureConfig) => {
-    InvoicePlugin.configure(config);
+    void InvoicePlugin.configure(config);
```

Comment:
Please us `.catch` here. Using void is dangerous:

When you call an async function, it returns a Promise. If you use void, it effectively discards that Promise.
This means any errors or rejections that occur inside the async function will be unhandled, which could lead to silent failures. Unhandled promise rejections will crash the Node Server